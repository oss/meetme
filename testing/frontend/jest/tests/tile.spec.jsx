import renderer from 'react-test-renderer';
import Tile from '../../../../frontend/components/utils/tile.jsx';

it('should render as expected and match the snapshot', () => {
    const container = renderer.create(<Tile />);
    expect(container).toMatchSnapshot();
});

//css files for branch coverage ideally we should re write the component
it('should render as expected and match the snapshot', () => {
    const container = renderer.create(<Tile title="Invite Your Collaborators" subtitle="this is a subtitle" grow default_padding={false} bg_color verticallyCenter default_margin={false} fullHeight overflowX extracss></Tile>);
    expect(container).toMatchSnapshot();
}); 
