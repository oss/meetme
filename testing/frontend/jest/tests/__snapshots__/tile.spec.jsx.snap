// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render as expected and match the snapshot 1`] = `
<div
  className="
       bg-white 
       px-8 py-5 
      shadow-sm
       
       mx-2 my-2 
      rounded-md 
      
       
       "
>
  <div
    className="flex justify-between items-center"
  >
    <div
      className="text-gray-600 font-bold"
    />
  </div>
</div>
`;

exports[`should render as expected and match the snapshot 2`] = `
<div
  className="
       true 
      
      shadow-sm
       flex flex-col justify-center 
       
      rounded-md  h-full 
       grow 
       overflow-x-auto 
       true"
>
  <div
    className="flex justify-between items-center"
  >
    <div
      className="text-gray-600 font-bold"
    >
      Invite Your Collaborators
    </div>
  </div>
  <div
    className="text-gray-500 text-sm"
  >
    this is a subtitle
  </div>
</div>
`;
